server/app/survey.go

```diff
@@ -214,10 +237,10 @@ func (a *UserSurveyApp) ensureSurveyBot() error {
 	return nil
 }
 
-func (a *UserSurveyApp) userInExcludedTeams(userID string, survey *model.Survey) (bool, error) {
+func (a *UserSurveyApp) userMemberOfFilteredTeams(userID string, survey *model.Survey) (bool, error) {
```

Comment:
Do we need to do anything to clear the cache for this method if the admin changes the new option to include or exclude users from a given team?

Reply:
The cache is per user, per survey (cache key uses survey ID and user ID), so, existing cache, or even cache from one survey, won't affect another survey. This also works fine if there are two surveys ending and starting within the span of cache expiry duration.